accepted/00000-snap-package-support.md

```diff
@@ -0,0 +1,175 @@
+- Feature Name: support snap in ubuntu 22.04
+- Start Date: 6/01/2025
+
+# Summary
+[summary]: #summary
+
+Introduce Snap support in Uyuni, allowing users to:
```

Comment:
This RFC itself details the reposync side, i.e. how do we:

- discover snaps
- select snaps for download
- download snaps
- store snaps

I'm missing the description of:
- The API, which will respond to requests from the minion's `snap` execution.
- The Salt side, which enables server to instruct clients to install/update/remove snaps.
  - For now, we can focus on using Salt directly, i.e. `salt <minion-name> snap.install <snap-name>` from the server means that the server reaches out to the minion, executes `snap install ...`, and finally minion uses the Server's new Snap API to install the snap.

We can either keep this in one RFC, of split it into smaller RFCs, that is up to you.